src/library/dlgtagfetcher.cpp

```diff
@@ -246,49 +306,105 @@ void DlgTagFetcher::fetchTagFinished(
     m_data.m_pending = false;
     m_data.m_results = guessedTrackReleases;
     // qDebug() << "number of results = " << guessedTrackReleases.size();
-    updateStack();
+    updateView();
 }
 
 void DlgTagFetcher::slotNetworkResult(
```

Comment:
```suggestion
void DlgTagFetcher::slotNetworkError(
```
is only called in the error case.